src/containers/screens/main/qr/QrCodeScanner.js

```diff
@@ -37,7 +37,7 @@ class QrCodeScanner extends Component {
     navigate = () => {
       const navigateToCreateOrRestore = NavigationActions.navigate({
         routeName: this.state.invoker,
-        params: { activeTab: 3 },
+        params: { activeTab: 1 },
```

Comment:
yea

Reply:
I don't think user flow would make sense in the following case:
1 .  Adding a new custom token (not from the database)
2.  Navigate to the qrscanner
3. upon a successful scan it would navigate to the first tab instead of the third tab(adding custom token page)

Reply:
@CFDevelopment @masterrom  I changed here beacause from addToken page , the qrScanner was navigating to first tab, while it should have been the third tab, But I don't how it might affect the contacts module. Please check if this change should be made

Reply:
@CFDevelopment @masterrom  Can you please check this flow, **Add token or coin** button to **new token** tab -> **qrScanner** and back, this navigates to 1st tab, rather than **new token** tab, if activeTab is set to 1. But this might be buggy in other modules. Thanks